src/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import MainContainer from 'src/MainContainer';
+import ErrorBoundary from 'Components/common/ErrorBoundary';
+import { bindActionCreators } from 'redux';
+import { requestApiData, clickStoreData } from 'Components/actions';
+import { connect } from 'react-redux';
+
+const App = () => (
+  <ErrorBoundary>
+    <MainContainer />
+  </ErrorBoundary>
+);
+
+const mapStateToProps = state => ({ movieReducer: state.movieReducer, params: [state.params]});
```

Comment:
didn't catch why you need this on App level, the same for mapdispatch

Reply:
Updated